server/views/lostBeds/index.njk

```diff
@@ -30,7 +30,7 @@
                 text: "Out of service"
               },
               value: {
-                text: numberOfLostBeds
```

Comment:
can we remove the numberOfLostBeds param passed into the view ?

Reply:
Good call!